src/coffea/dataset_tools/preprocess.py

```diff
@@ -101,13 +118,22 @@ def _get_steps(
                 "object_path": arg.object_path,
                 "steps": out_steps,
                 "uuid": out_uuid,
+                "form": form_json,
+                "form_hash": form_hash,
```

Comment:
Having the hash algo in the name would be useful (e.g. `form_hash_md5` or `form_hash_sha512` etc)

Reply:
will do!